APP/ev_chargers/lib/screens/homepage.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:ev_chargers/style.dart';
+import 'package:flutter/material.dart';
+import 'map_page.dart';
+import 'reservation_page.dart';
+import 'profile_page.dart';
+import 'settings_page.dart';
+
+class MyHome extends StatefulWidget {
+  const MyHome({super.key});
+
+  @override
+  MyHomeState createState() => MyHomeState();
+}
+
+class MyHomeState extends State<MyHome> with SingleTickerProviderStateMixin {
+  late TabController _tabController;
+  static const List<Tab> navigationTabs = <Tab>[
+    Tab(icon: Icon(Icons.person_outline),),
+              Tab(icon: Icon(Icons.calendar_today_outlined),),
+              Tab(icon: Icon(Icons.map_outlined),),
+              Tab( icon: Icon(Icons.settings_outlined),),
+  ];
+  List<Widget> pages = const [ProfilePage(),ReservationPage(),MapPage(), SettingsPage()]; 
```

Comment:
I wouldn't say this is properly formatted.

Reply:
Prettier corrects me to this, idk